keyboards/handwired/hillside/52/readme.md

```diff
@@ -0,0 +1,29 @@
+# Hillside 52
+
+![hillside](https://imgur.com/erJPjvr.png)
```

Comment:
```suggestion
![hillside](https://imgur.com/erJPjvrh.png)
```

Reply:
Hmm. They both exist with .png as images, but only 
https://imgur.com/erJPjvr   exists as a page. So I think 
https://imgur.com/erJPjvrh was an earlier deleted attempt at putting images up.

Their similarity makes me suspicious though. 

How did you even find the h one?

Reply:
it's a trick of imgur.  adding l or h to the file name will show a lower resolution version of the existing image.  And one that renders much better in the readme files.

Reply:
Thanks. There are other links in the keymap as well. Let me update them.